server/game/cards/02_SHD/units/OmegaPartOfTheSquad.ts

```diff
@@ -48,7 +48,7 @@ export default class OmegaPartOfTheSquad extends NonLeaderUnitCard {
             playedCardEntry.card.hasSomeTrait(Trait.Clone) &&
             playedCardEntry.card !== card
         );
-        return clonesPlayedByThisPlayerThisPhase.length === 0;
+        return card.hasSomeTrait(Trait.Clone) && clonesPlayedByThisPlayerThisPhase.length === 0;
```

Comment:
I think you should test card has clone trait before iterating watcher to improve performance

Reply:
Done